account_invoice_inter_company_queued/models/account_move.py

```diff
@@ -8,7 +8,10 @@ class AccountMove(models.Model):
     def _post(self, soft=True):
         self = self.with_context(account_invoice_inter_company_queued=True)
         res = super(AccountMove, self)._post(soft=soft)
-        dest_company = self._find_company_from_invoice_partner()
-        if dest_company and not self.auto_generated:
-            self.sudo().with_delay().create_counterpart_invoices()
+        if self:
+            dest_company = self._find_company_from_invoice_partner()
```

Comment:
@ilyasProgrammer we got an issue if you have multiple invoice to post, the error is raised because of this:

https://github.com/OCA/multi-company/blob/b2714b1d849f06de60d7f2316e80f99d2c96b353/account_invoice_inter_company/models/account_move.py#L55-L62

function expert a singleton, so it's needed to loop over the records in order to fix it.

Reply:
fixed

Reply:
thx, can rebase against 14.0 (just in case)